### PR TITLE
Role assignment

### PR DESCRIPTION
**Pull request by timreimherr**
@wojonet 
I believe this is the most up to date branch. The branches got tangled but I think it's all cleared up now.

### CODE REVIEW

**Comment by timreimherr** [comment]
@wojonet resolved merge conflicts

File: Blog/GamingGuruBlog.Web/GamingGuruBlog.Domain/BlogServices.cs [hunk 1]
```diff
@@ -9,24 +9,33 @@ namespace GamingGuruBlog.Domain
     public class BlogServices : IBlogServices
     {
         private IBlogPostRepository _blogPostRepo;
-        private ICategoryRepository _categoryRepo;
         private IUserRepository _userRepo;
+<<<<<<< HEAD
```
Comment (wojonet) [inline Blog/GamingGuruBlog.Web/GamingGuruBlog.Domain/BlogServices.cs]: Looks like you missed a merge

**Comment by timreimherr** [comment]
@wojonet fixed merge conflicts

**Comment by wojonet** [comment]
looks good

**Pull request merged**